### PR TITLE
[ADD] connector_voicent

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#186 

Voicent Connector
==============
This module allows you to connect Odoo with Voicent <https://www.voicent.com> and is meant to be extended to integrate Odoo records and processes with phone calls made by Voicent.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can you please review this PR.

File: connector_voicent/models/backend_voicent.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2019 Open Source Integrators
+# <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+import datetime
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from pytz import timezone
+from odoo.addons.connector_voicent.examples import voicent
```
Comment (max3903) [inline connector_voicent/models/backend_voicent.py]:
```suggestion
from .examples import voicent
```

File: connector_voicent/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
+* Youness MAAFI <youness.maafi@gmail.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (max3903) [inline connector_voicent/readme/CONTRIBUTORS.rst]:
```suggestion
* Murtuza Saleh <murtuzasaleh@serpentcs.com>
```

File: oca_dependencies.txt [hunk 3]
```diff
@@ -1 +1,4 @@
-server-tools
\ No newline at end of file
+server-tools
+connector
+queue
+
```
Comment (max3903) [inline oca_dependencies.txt]:
```suggestion
```

Please sort the file alphabetically.

**Comment by max3903** [comment]
@murtuzasaleh Looks good. I will run some tests next week.

**Comment by murtuzasaleh** [comment]
@max3903  


1) I don't think we need this condition because it is worked based on next call so when the job is created pass the Next Call value in `eta` so the job is to execute based on that.



2) When the new job is created how we can pass the date in `eta` field for the execution of this code because we can not pass Next Call field value on it, and job is execution part does not depend on the Scheduler, we can delay it based on `eta` and pass it where the method is defined.

Correct me if I am wrong.

**Comment by max3903** [comment]
@murtuzasaleh 

1. Correct.
2. The job is created with the link to the backend (where the next call value is stored) so the job knows at what time it should be executed.

**Review by max3903 — APPROVED** [review]

**Review by patrickrwilson — APPROVED** [review]

**Pull request merged**